configure.ac

```diff
@@ -1719,6 +1719,17 @@ esac
 AC_SUBST(UNITTEST_CFLAGS)
 AC_SUBST(UNITTEST_LDFLAGS)
 
+dnl do not add -lresolv for freebsd
+case $host_os in
+  freebsd*)
+    LRESOLV_FLAG=""
+    ;;
+  *)
+    LRESOLV_FLAG="-lresolv"
```

Comment:
Probably not the best name. Something like `RESOLVE_LIBS` would be more meaningful and similar to other variables used for the same purpose.